add pull request template (#21)
inbo/protocols: .github/pull_request_template.md

<!--- indicate the protocol code in the Title above -->

## Task list

<!--see https://docs.github.com/en/github/managing-your-work-on-github/about-task-lists
for an explanation on how to use task lists-->

Steps by contributor:

- [ ] Add description to this pull request (see #-description)
- [ ] Submit the pull request by clicking 'create pull request'

> - [ ] Mark the pull request as draft

Author: Didn't know that, but will leave it as is.

- [ ] Check if the version number in the `index.Rmd` yaml section is of format `yyyy.nn.dev`
- [ ] In case of a protocol created from a pre-existing `docx` protocol, check if all sections comply with current template for a new protocol
- [ ] Add further commits and push commits to GitHub if needed
- [ ] Update the protocol-specific `NEWS.Rmd`
- [ ] Mark the pull request as 'ready for review'

Review steps:

- [ ] Add reviewers, at least one subject-matter specialist and one administrator
- [ ] Wait for review comments and address them
- [ ] Iterate until reviewer approvals (merging the pull request will be done by an administrator)

To be done by an administrator after review, but before merging the pull request:

- [ ] Update the `.zenodo.json` file (authorship will extend as protocols are added)
- [ ] Bump the version number in the `index.Rmd` yaml section from `yyyy.nn.dev` to `YYYY.NN`.
- [ ] Update the repo `NEWS.md` file

To be done by an administrator after merging this pull request:

- [ ] Add general tag to the merge commit (`protocols-YYYY.NN`)  (see [release model](README.md#release-model))

> - [ ] Add specific tag to the merge commit (`protocol-code-YYYY.NN`)  (see [release model](README.md#release-model))

Reviewer: ```suggestion
- [ ] Add specific tag to the merge commit (`<protocol-code>-YYYY.NN`)  (see [release model](README.md#release-model))
```

I suggest to do this in README as well. `protocol-code` is a variable, whereas in `protocols-YYYY.NN` it is a string, which is confusing to the new reader. I know, this also holds for YYYY and NN, but I think this is more clear.

> - [ ] Check if continuous integration proceeded without problems

Reviewer: ```suggestion
- [ ] Check whether continuous integration proceeded without problems
```

> - [ ] Go to releases and create a new release based on a general tag

Reviewer: ```suggestion
- [ ] [Create a new release](https://github.com/inbo/protocols/releases/new) based on a general tag
```

> - [ ] Check success of publication steps at protocols.inbo.be and at Zenodo (records both source and rendered) 

Reviewer: If this address 'protocols.inbo.be' is correct, then best update it in the general repo documentation.

Also, best provide it here as a hyperlink, it is now plain text.

To do once a first Zenodo release has been made: make it a hyperlink here (with the version-agnostic DOI). You can neglect this now, we'll rediscover that.

Author: The url is not yet 100% sure.

## Description

> <!--- Describe your protocol proposal -->

Reviewer: ```suggestion
<!--- Describe your protocol proposal or your proposed changes-->
```

<!--- indicate the protocol code in the Title above -->

## Task list

<!--see https://docs.github.com/en/github/managing-your-work-on-github/about-task-lists
for an explanation on how to use task lists-->

Steps by contributor:

- [ ] Add description to this pull request (see #-description)
- [ ] Submit the pull request by clicking 'create pull request'
- [ ] Mark the pull request as draft
- [ ] Add further commits and push commits to GitHub if needed

> - [ ] Update the protocol-specific NEWS.Rmd

Reviewer 2: ```suggestion
- [ ] Update the protocol-specific NEWS.md
```

Author: It's an Rmd file [see template](https://github.com/inbo/protocolhelper/tree/master/inst/rmarkdown/templates/template_sfp_nl/skeleton)

Reviewer 2: Can you remind my why we have it as an Rmd? If it doesn't contain R code, i'd use an md. Because that can be rendered by GitHub.
